BrainPortal/app/views/data_providers/new.html.erb

```diff
@@ -0,0 +1,261 @@
+
```

Comment:
I see two problems with this new file:

* the file `form_fields.html.erb` is no longer needed
* more importantly, much of the embedded doc is obsolete, and a fixed version of it is already in another pull request #393 (installed in a separate file).